### PR TITLE
Help pages

### LINKED ISSUE #8: Add Help pages
Partner says she has nearly finished them. Will perhaps send them one by one, or as one big batch

### PR DESCRIPTION
**Pull request by lizconlan**
Interim review requested to check whether I've got anything badly wrong. ToDos in the ticket

Fixes #8

### CODE REVIEW

File: lib/views/help/officers.html.erb [hunk 1]
```diff
@@ -1,245 +1,115 @@
...
-must respond <strong>promptly</strong> to FOI requests. If they fail to do that,
-it is best if they show the hard work they are doing by explaining what is
-taking the extra time to do.
-</p>
-
-<p>That said, <%= site_name %> does show the maximum legal deadline
-for response on each request. Here's how we calculate it.</p>
-
-<ul>
+  <p>Para que el sistema funcione de manera automática, utilizamos direcciones de correo electrónico generadas por ordenador para cada una de las solicitudes. Es por eso que la dirección de contacto que a usted le aparece pone “@datasketch.co”.</p>
```
Comment (garethrees) [inline lib/views/help/officers.html.erb]: Wouldn't this be `@queremosdatos.co`?

**Comment by garethrees** [comment]
Looks good on the whole.

Might be worth changing `mailto:queremosdatos@datasketch.co` to a link to the contact form, but that obviously doesn't exist yet.

File: lib/views/help/officers.html.erb (hunk 1 above)
Comment (lizconlan) [reply lib/views/help/officers.html.erb]: I think I'll change that to `<%= AlaveteliConfiguration::incoming_email_domain %>` as per the belgium theme

**Comment by lizconlan** [comment]
Have left the `mailto:` links for now but changed them to use the `@contact_email` (set from config) so that they can set in one central place

**Comment by garethrees** [comment]
All looks fine; best to get feedback from Colombia team anyway 👍

**Pull request merged**